src/lib/rest/rest.cpp

```diff
@@ -704,6 +704,13 @@ int servicePathCheck(ConnectionInfo* ciP, const char* servicePath)
     return 1;
   }
 
+  if (servicePath[1] == '/')
+  {
+    OrionError oe(SccBadRequest, "Zero length service path levels are not allowed, minimum 1 charater is required");
+    ciP->answer = oe.setStatusCodeAndSmartRender(ciP->apiVersion, &(ciP->httpStatusCode));
+    return 1;
+  }
```

Comment:
Not sure if this is going to work if 0-length service path level is in the middle of the service path strings...

Maybe the check should be done within the `for (int ix = 0; ix < components; ++ix)` loop? Not sure...

Reply:
I have just seen you comment here https://github.com/telefonicaid/fiware-orion/issues/4495#issuecomment-1929216781, so it seems it will be already covered :)

Reply:
However, for homogeneity, I'd suggest to use the same error message in both cases:

```
"empty component in ServicePath"
```

Reply:
fixed in f67c041